### ISSUE TITLE
LaTeX: preserve newlines for enumerations

### ISSUE THREAD

**Issue by nvanbenschoten** [stkb/vscode-rewrap#24]
First off, thanks for this awesome plugin! It is exactly what I've been looking for for a while.

I just began using Rewrap to edit LaTex. It works great for almost all cases, but I noticed one issue. When wrapping enumerations, Rewrap will try to wrap multiple items into the same row. For instance, it will make the transformation from:

```
\begin{enumerate}
    \item $\neg aRa$ (irreflexive)
    \item $aRb ^ bRa \Rightarrow a = b$ (antisymmetric)
    \item $aRb ^ bRc \Rightarrow aRc$ (transitive)
\end{enumerate}
```

to

```
\begin{enumerate}
    \item $\neg aRa$ (irreflexive) \item $aRb ^ bRa \Rightarrow a = b$ (antisymmetric) \item $aRb ^
    bRc \Rightarrow aRc$ (transitive)
\end{enumerate}
```

I'd expect the plugin to respect the newlines for each item. I saw that you recently added LaTeX support in v1.1.0, so this is probably a result of how new the support is.

**Comment by stkb** [comment]
Thanks for the feedback; I'll get working on fixing it!

**Comment by stkb** [comment]
Sorry it took a little while. Have just published [v1.2.0](https://github.com/stkb/vscode-rewrap/releases/tag/v1.2.0) where this should be fixed, as well as a couple of other improvements for LaTeX. I'd say there's still some work to do so let me know if you come across any other problems.

**Comment by stkb** [comment]
Closing as I think this is now fixed. Let me know if you're still having issues!
